virus/Partitivirus_rumenrex.md

```diff
@@ -0,0 +1,484 @@
+
+The third, alternative, but related, hypothesis is that the virus’s primary hosts are the resident anaerobic rumen fungi themselves (such as *Neocallimastix* or *Piromyces*), and that the vegetation simply acts as a vehicle bringing in carbon and occasional new fungal strains rather than serving as the main viral reservoir. In both scenarios, the key point is that the virus is associated with fungi, either plant-associated or rumen-resident, and the rumen provides the ideal ecological and physiological conditions for these fungal hosts, which explains why the virus is consistently observed there rather than in other parts of the digestive tract.
+
+To distinguish whether the virus primarily infects plant or feed associated fungi versus resident rumen fungi, I first determined what I would do in the wet-lab as this is my background. Thus, I would (1) quantify viral load in feed and rumen fungal fractions by virus-specific RT-qPCR, (2) culture fungi from both feed and rumen and screen individual isolates for the virus, and (3) where possible, perform time-course experiments following a switch from virus-negative to virus-positive feed. Higher viral prevalence in feed fungi, with transient rumen detection, would support a feed-associated endophyte host, whereas strong enrichment in rumen anaerobic fungi, especially after feed changes, would support a rumen-resident fungal host.
+
+However, to provide some preliminary bioinformatic support for either hypothesis I decided to create a phylogenetic tree using MUSCLE to place my virus in an evolutionary context [3]. The figure below also contains some samples from the initial BLAST projects that I found through my BioProjects [1,10]. New Zealand sheep refers to the methane production in sheep study, and China and Germany Cattle also refer to the previously mentioned studies that had my virus present in the rumen of these animals between a 50-60% identity [22,19,27].
+
+![](img/Partitivirus_rumenrex/Phylo1.png)
```

Comment:
Give each figure a title between the square brackets. This displays if the image does not render.